cypress/integration/circulation-log/search.spec.js

```diff
@@ -39,7 +42,7 @@ describe('ui-circulation-log', () => {
             });
           })
           .then(() => {
-            cy.addServicePointToUser(Cypress.env('servicePoints')[0].id, userId);
+            cy.addServicePointToUser([Cypress.env('servicePoints')[0].id], userId);
```

Comment:
1. use the same method from cypress\support\fragments\users\usersEditPage.js. After this remove  cy.addServicePointToUser from cypress\support\api\servicePoints.js
2. method is addServicePointToUser. It means you can use it to add 1 service point. I suggest create separated method to several service points adding. In case of 1 service point adding - you need only 2 parameters - userId and servicePointId(which is default to user)

Reply:
or rename addServicePointToUser to addServicePointsToUser

Reply:
Done